blueprints.yaml

```diff
@@ -71,19 +82,7 @@ form:
                   label: PLUGIN_GANALYTICS.OBJECT_NAME
                   help: PLUGIN_GANALYTICS.OBJECT_NAME_HELP
                   size: small
-                  default: "ga"
-
-                async:
-                  type: toggle
-                  label: PLUGIN_GANALYTICS.ASYNC
-                  help: PLUGIN_GANALYTICS.ASYNC_HELP
```

Comment:
**Question:** Are these constants still used elsewhere? If not, should be removed